client/widgets/test-execution-time/time.directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('probedock.testExecutionTimeWidget').directive('testExecutionTimeWidget', function() {
+  return {
+    restrict: 'E',
+    controller: 'TestExecutionTimeWidgetCtrl',
+    templateUrl: '/templates/widgets/test-execution-time/time.template.html',
+    scope: {
+      organization: '=',
+      test: '='
+    }
+  };
+}).controller('TestExecutionTimeWidgetCtrl', function(api, $scope, projectVersions) {
+
+  var page = 1;
+
+  $scope.pageSize = 25;
+  $scope.maxResults = 100;
+
+  $scope.params = {};
+  $scope.results = [];
+  $scope.loading = true;
+
+  $scope.chart = {
+    data: [],
+    labels: [],
+    options: {
+      pointHitDetectionRadius: 5,
+      showTooltips: false,
+      pointDot: false,
+      datasetFill: false,
+      scaleShowVerticalLines: false,
+
+      /*
+       * Fix for space issue in the Y axis labels
+       * see: https://github.com/nnnick/Chart.js/issues/729
+       * see: http://stackoverflow.com/questions/26498171/how-do-i-prevent-the-scale-labels-from-being-cut-off-in-chartjs
+       */
+      scaleLabel: function (object) {
+        return '  ' + object.value;
```

Comment:
See if you can use [this filter](https://github.com/probedock/probedock/blob/master/client/filters/formatDuration.filter.js) to format the value. It's quite configurable (although it lacks documentation). The `maxParts` argument should avoid an overly long string if set to 1. You shouldn't need the last two.